src/hotspot/cpu/s390/gc/g1/g1BarrierSetAssembler_s390.cpp

```diff
@@ -214,6 +214,9 @@ void G1BarrierSetAssembler::g1_write_barrier_pre(MacroAssembler* masm, Decorator
 
   __ bind(callRuntime);
 
+  // TODO: GPRS: 5 + FPRS: 8 + Flags:1?
+  const int nbytes_save = (5 + 8 + 1) * BytesPerWord;
```

Comment:
Flags are not needed.

Reply:
I thought you might say that. This has been corrected.